framework/src/modules/chain/transport/transport.js

```diff
@@ -366,6 +365,13 @@ class Transport {
 			};
 		}
 
+		if (ids.length > this.constants.maxSharedTransactions) {
+			return {
```

Comment:
it would be probably nice to have comments here to address in the banning mechanism.